gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql

```diff
@@ -0,0 +1,64 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql */
+
+/*
+ * Returns columns (table_schema, table_name) ordered by increasing vacuum time.
+ * In this list, if arg is true, then tables that are not yet vacuumed are located first,
+ * and already vacuumed - at the end, else (arg is false) tables that are already
+ * vacuumed are located first, and tables that are not yet vacuumed are located at the end.
+ */
+CREATE FUNCTION arenadata_toolkit.adb_vacuum_strategy_arg(actionname TEXT, arg BOOLEAN)
+RETURNS TABLE (table_schema NAME, table_name NAME) AS
+$func$
+BEGIN
+	RETURN query EXECUTE format($$
+	SELECT n.nspname AS table_schema, c.relname AS table_name
+	FROM pg_class c
+	JOIN pg_namespace n ON c.relnamespace = n.oid
+	LEFT JOIN pg_stat_last_operation o ON o.classid = 'pg_class'::regclass::oid
+		AND o.objid = c.oid AND o.staactionname = UPPER(%L)
+	LEFT JOIN pg_partition_rule p ON p.parchildrelid = c.oid
+	WHERE c.relkind = 'r'
+		AND c.relstorage != 'x'
+		AND n.nspname NOT IN (SELECT schema_name FROM arenadata_toolkit.operation_exclude)
+		AND p.parchildrelid IS NULL
+		ORDER BY statime ASC NULLS %s
```

Comment:
"ORDER BY" should be at the same level as WHERE, because it is not a part of WHERE

Reply:
moved